sdcm/cluster.py

```diff
@@ -3026,23 +3042,26 @@ def install_scylla_monitoring_prereqs(self, node):
                 pip install pyyaml
             """)
         elif node.is_debian():
+            node.remoter.run('sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F76221572C52609D', retry=3)
+            node.remoter.run(cmd="sudo apt-get update", ignore_status=True)
+            node.remoter.run(cmd="sudo apt-get dist-upgrade -y")
+            node.remoter.run(
+                cmd="sudo echo 'deb https://apt.dockerproject.org/repo debian-jessie main' | sudo tee /etc/apt/sources.list.d/docker.list")
```

Comment:
It only works for debian 8

Reply:
right, these are only Debian-8 adjustments since Manager support Jesse only.

can you please tell if we have SCT test coverage for Monitoring with Debian-9?

Reply:
changed to "node.is_debian8()"